Update API Sync tasks based on RC features (#81)
mulesoft/docs-studio: modules/ROOT/pages/import-api-specification-local-file.adoc

> = Importing an API Specification from a Local File

Reviewer: DITTO GLOBALs re. wording, order consistency throughout:
```suggestion
= Import an API Specification from a Local File
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

Anypoint Studio allows you to import an API specification from your local machine and import it either into a new Mule Project, or an existing one.

> This task only works for RAML types of API specifications.

Reviewer: DITTO edits to same text earlier in topic set:

I suggest moving this point into the intro, by specifying RAML there (if we mean the same thing by "specification" as by "definition":

"Anypoint Studio enables you to import a RAML API definition from Design Center into either a new Mule project or an existing one."

Perhaps we could add the size limitation here as well:

"Anypoint Studio enables you to import a RAML API definition of fewer than 4,000 lines from Design Center into either a new Mule project or an existing one."

Just ideas....

> . In the top task bar, select *File*, then select *New*, and then *Mule Project*.

Reviewer: ```suggestion
. In the taskbar at the top of the Anypoint Studio display, select *File* > *New* > *Mule Project*.
```

. Type a name for your project.

> . Select a Mule runtime version.

Reviewer: ```suggestion
. Select a Mule runtime engine version.
```
No restrictions, as before?

. In the *API Specification* section, select the *From Design Center or Local Files* tab.
. In the *Location* field, type the location of your local API, or click the *...* button and select *Browse Files...*.

> You can import a RAML file, or an entire zip file bundle if your API has schemas and examples.

Reviewer: ```suggestion
You can import either a RAML file or an entire .zip file bundle if your API has schemas and examples.
```

. Click *Finish*.

> Studio scaffolds your API specification. If you choose not to create flows out of your API specification, uncheck the *Scaffold Flows From These API Specifications* option.

Reviewer: This task topic is titled "Importing," so is it confusing to use "scaffolds" as the result of this task?
Also, at what point does the user unselect the Scaffold Flows checkbox? 
Should this step be in the previous task?

If this option is really a checkbox...:

```suggestion
Studio scaffolds your API specification. If you choose not to create flows out of your API specification, unselect the *Scaffold Flows From These API Specifications* checkbox.
```
THIS IS A GLOBAL EDIT to each topic that contains this text.

> You can import more than one API. You can select the API you want to scaffold flows and check the *Scaffold selected API specifications* checkbox so that Studio generates the flow based on the imported API.

Reviewer: The ability to import more than one API should probably be in the intro text, and selecting the API to scaffold and checking the checkbox sounds like it should be a step in the task steps above.

> It is not possible to add a local API specification to an existing Mule project.

Reviewer: This sounds like something that belongs in the intro text, so that readers proceed no further if they want to do this....

Also...is this here only until the following task is validated?

// VALIDATE
// == Importing a Local API Into an Existing Project

> //

Reviewer: DITTO consider text transition between title and first step if there is something useful to add about why our audience might want to choose this method over creating a new project. GLOBAL in these topics.

> // . Right click on your project in the Package Explorer view, then select *New*, and then *RAML API Definition*.

Reviewer: ```suggestion
// . In the Package Explorer view, right-click on your project and select *Mule* > *RAML API Definition*.
```

> // . In the *File Name* type the name for your API specificaton.

Reviewer: ```suggestion
// . In the *File Name* field, type a name for your API specification.
```

> // . Click the *<< Advanced* button, and check the *Link to File in the Filesystem*.

Reviewer: ```suggestion
// . Click the *<< Advanced* button and select the *Link to File in the Filesystem* checkbox.
```
